README.md

```diff
@@ -63,45 +89,46 @@ curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/master/deployments/sc
 curl "https://github.com/ZupIT/horusec/releases/latest/download/horusec_win_x64.exe" -o "./horusec.exe" && ./horusec.exe version
 ```
 
-All binaries with versions can be found in our [releases page](https://github.com/ZupIT/horusec/releases).
+- You can find all binaries with versions in our [**releases page**](https://github.com/ZupIT/horusec/releases).
 
-Click [here](https://horusec.io/docs/cli/installation) to see more details in how to install.
+- For more details on how to install, check out the [**documentation**](https://horusec.io/docs/cli/installation) 
 
-#### Check the installation
+#### **Check the installation**
 ```bash
 horusec version
 ```
 
-## Usage CLI
-To use horusec-cli and check the application's vulnerabilities:
+## **Usage**
+### **CLI Usage**
+To use horusec-cli and check the application's vulnerabilities, use the following command:
 ```bash
 horusec start -p .
 ```
+{{% alert color="warning" %}}
+When horusec starts an analysis, it creates a folder called **`.horusec`**. This folder is the basis for not changing your code. We recommend you to add the line **`.horusec`** into your **`.gitignore`** file so that this folder does not need to be sent to your git server.
+{{% /alert %}}
```

Comment:
alert not work in github markdown 
please change to: 
```suggestion
> When horusec starts an analysis, it creates a folder called **`.horusec`**. This folder is the basis for not changing your code. We recommend you to add the line **`.horusec`** into your **`.gitignore`** file so that this folder does not need to be sent to your git server.
```